routes/tasks.go

```diff
@@ -0,0 +1,167 @@
+package routes
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jinzhu/gorm"
+	"github.com/labstack/echo"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/kirikami/go_exercise_api/database"
+)
+
+func SaveTaskHandler(c echo.Context) error {
+	const timeForm = "3 04 PM"
+	var ids int
+	var id int64
+
+	currentTime := time.Now()
+	tasks := []database.Task{}
+
+	db := c.Get("DBConnection").(*gorm.DB)
+
+	if c.FormValue("id") != "" {
```

Comment:
Move entity id validation into helper function `ParseIdInt64FromString() (int64, error)`, you will use this a lot in every handler. And use int64 for IDs